release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,932 @@
+resize will be added with RHEL 7.5.
+
+. Add `AllowVolumeExpansion=true` to the storage class.
+. Run:
++
+----
+$ oc edit pvc claim-name
+----
+
+. Edit the `storage` field, which is under `spec` and `requests`, with the new value.
+
+[[ocp-container-native-storage-glusterfs-pv-consumption-metrics-available]]
+==== Container Native Storage GlusterFS PV Consumption Metrics Available from {product-title}
+
+Container Native Storage GlusterFS is extended to provide volume metrics
+(including consumption) through Prometheus or Query.
+
+Metrics are available from the PVC endpoint. This adds visibility to what is
+being allocated and what is being consumed. Previously, you could only see
+allocated size of the PVs. Now, you know how much is really consumed so, if
+needed, you can expand it before it runs out of space. This also allows
+administrators to do billing based on consumption, if needed.
+
+Examples of added metrics include:
+
+* `kubelet_volume_stats_capacity_bytes`
+* `kubelet_volume_stats_inodes`
+* `kubelet_volume_stats_inodes_free`
+* `kubelet_volume_stats_inodes_used`
+* `kubelet_volume_stats_used_bytes`
+
+[[ocp-3-9-cns-now-supports-custom-volume-naming]]
+==== CNS Now Supports Custom Volume Naming
+
+You can now specify custom volume names (prefixes) for PVs from the CNS backed
+storage class.
+
+This is now an improved naming convention of allocating volumes so that you can
+enforce back-end data management and data protection policies. Now with CNS, you
+can specify prefix. Even in dynamic allocation of PVs, PV names will be a
+combination of volume prefix, project name, namespace, claim name, and UUID,
+allowing you to easily identify the PVs, get context, and apply policies.
+
+[[ocp-3-9-automated-cns-deployments-with-advanced-installation]]
+==== Automated CNS Deployment with {product-title} Advanced Installation
+
+In the {product-title} advanced installer, the CNS block provisioner deployment
+is fixed and the CNS Un-install Playbook is added. This resolves the issue of CNS
+block deployment with {product-title} and also provides a way to uninstall a failed
+installation of CNS.
+
+CNS storage device details are added to the installer’s inventory file. The
+advanced installer manages configuration and deployment of CNS, file and block
+provisioners, registry, and ready-to-use PVs.
+
+[[ocp-39-scale]]
+=== Scale
+
+[[ocp-39-scale-cluster-limits]]
+==== Cluster Limits
+
+Updated guidance around
+xref:../scaling_performance/cluster_limits.adoc#scaling-performance-cluster-limits[Cluster
+Limits] for {product-title} 3.9 is now available.
+
+[[ocp-39-device-plugins]]
+==== Device Plug-ins
+
+Device plug-ins allow you to use a particular device type (GPU, InfiniBand,
+or other similar computing resources that require vendor-specific initialization
++nd setup) in your {product-title} pod without needing to write custom code. The
++evice plug-in provides a consistent and portable solution to consume hardware
```

Comment:
@ahardin-rh I think there's a couple extra pluses at the start of the above two lines. Maybe something errant from a copy/paste?